src/main/java/uk/gov/hmcts/reform/prl/controllers/hearingmanagement/HearingsManagementController.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.prl.controllers.hearingmanagement;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
```

Comment:
Functional integration and unit tests are missing

Reply:
Functional test cannot be performed at this stage, as we need to integrate this API with fis_hmc_api. Unit tests are added in the following branch: hearing-api-demo-deploy

Reply:
https://github.com/hmcts/prl-cos-api/pull/746